src/libraries/Lockers.sol

```diff
@@ -58,7 +66,16 @@ library Lockers {
     }
 
     function getLocker(uint256 i) internal view returns (address locker) {
-        uint256 slot = LOCKERS_SLOT + i;
+        // first slot of the ith array item
+        uint256 slot = LOCKERS_SLOT + (i * LOCKER_STRUCT_SIZE);
+        assembly {
+            locker := tload(slot)
+        }
+    }
+
+    function getLockOriginator(uint256 i) internal view returns (address locker) {
```

Comment:
name suggestions appreciated! I'm currently thinking maybe we should call this `locker` and call the other `lockTarget`

Reply:
yeah heh I was confused by the name. so there are two addrs now
- the address that implements the lockCallback (this is the locker) ?
- the address that triggers the lock call (this is the lockCaller) ?

Reply:
thats the best i could come up w though 😄

Reply:
ahh lockCaller is better